contracts/ssnlist.scilla

```diff
@@ -641,16 +659,25 @@ procedure ReDelegStakeAmt(initiator: ByStr20, ssn: ByStr20, redeleg_amt: Uint128
       HasBufferedDeposit ssn initiator;
       AdjustDeleg ssn initiator amt redeleg_amt;
       (* Illegal withdraw amount could be handled in procedure AdjustDeleg *)
-      DecreaseTotalStakeAmt redeleg_amt;
       new_amt = builtin sub stake_amt redeleg_amt;
       minstake_tmp <- minstake;
       status = uint128_le minstake_tmp new_amt;
       ssn_option_tmp = Ssn status new_amt rewards name urlraw urlapi buffdeposit comm comm_rewards rec_addr;
       ssnlist[ssn] := ssn_option_tmp;
-      (* If ssn becomes inactive, we need to decrease the rest ones also *)
-      DecreaseTotalStakeAmtOnStatus new_amt status;
       e = { _eventname: "Remove delegate from SSN"; ssn_addr: ssn; deleg_address: initiator; amt: redeleg_amt };
-      event e
```

Comment:
Why change in `ReDelegStakeAmt`? This is not part of findings in 6.1? Can we move it as a separate PR?